src/components/Filters/components/ConnectedFilterControl/ConnectedFilterControl.scss

```diff
@@ -78,8 +79,21 @@ $stacking-order: (
   }
 }
 
+.RightContainer.queryFieldHidden {
+  .Item:first-of-type > div > button {
+    border-top-left-radius: var(
+      --p-border-radius-base,
+      border-radius()
+    ) !important;
```

Comment:
With all of this specificity do we still need `!important`?

Reply:
Good suggestion! 
Had to change a few of the selectors but was able to remove all of the `!important` selectors 😄